### PR TITLE
CH-22 but actually CH-36

### PR DESCRIPTION
**Pull request by Jpena91**
-modified createRosterDateMap to accept offset
-added next/last week function
-THIS IS ACTUALLY CH-36

### CODE REVIEW

File: commands/last.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const nbaUtil = require('../util/nbaUtils')
+const yUtils = require('../util/yahooUtils')
+
+module.exports = {
+  name: 'last',
```
Comment (shibachurro) [inline commands/last.js]: should we rename this command to prev or PreviousWeek? Based of the description, last feels more like getting the last week of the schedule

File: commands/last.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  description: 'returns user roster for previous of current week',
+  async execute (message, args) {
+    let userRoster = await yUtils.createRosterDateMap(-6).then(resp => (resp))
+    //-6 to account for default Sunday start and -6 to get to last monday
```
Comment (shibachurro) [inline commands/last.js]: was this supposed to say // 0 to account for default Sunday start and -6 to get to last monday?

**Review by shibachurro — COMMENTED** [review]
Just wanted to comment for other reviewers, recall that Moment index for day() starts at 0 for Sunday

File: commands/last.js (hunk 1 above)
Comment (Laturine) [reply commands/last.js]: I vote for 'previous'

Comment (Jpena91) [reply commands/last.js]: Changed to previous

**Comment by shibachurro** [comment]
Looks good lets merge to master

**Comment by Jpena91** [comment]
I ran into a weird issue with Sunday checking. Since it thought week started on Sunday, it would give me what I would expect to be the next week since I'm treating Sunday as end of week instead. So if user checks roster on Sunday, I added a shift to 'previous' week's Monday which would include this current Sunday. Same rules apply for 'previous' and 'next' commands

**Comment by Jpena91** [comment]
> Looks good lets merge to master

Gonna hit the button now

**Pull request merged**